### PR TITLE
Sentence-based line breaking in operatorsandexpressions.tex

### PR DESCRIPTION
**Pull request by henrikt-ma**
Slowly, one by one…  Reminded by https://github.com/modelica/ModelicaSpecification/pull/3025#discussion_r748393274, I thought it was about time we convert also this chapter to sentence-based line breaks.

To be merged after #3025 in order to not introduce merge conflicts.

As the conversion is done by hand, this commit also includes a small number of other fixes of cleanup character.

### CODE REVIEW

File: chapters/operatorsandexpressions.tex [hunk 1]
```diff
@@ -1235,7 +1235,8 @@ \subsection{Event-Related Operators with Function Syntax}\label{event-related-op
 equation
   x = if time < 1 then 2 else time - 2;
   z = smooth(0, if time < 0 then 0 else time);
-  y = smooth(1, noEvent(if x < 0 then 0 else sqrt(x) * x)); // noEvent is necessary.
+  y = smooth(1,
```
Comment (HansOlsson) [inline chapters/operatorsandexpressions.tex:1238]: This seems somewhat spurious.
Was the line too long?

**Review by HansOlsson — APPROVED** [review]
Looks good (except for the merge-conflict).

File: chapters/operatorsandexpressions.tex (hunk 1 above)
Comment (henrikt-ma) [reply chapters/operatorsandexpressions.tex:1238]: Yes.

**Comment by henrikt-ma** [comment]
> Looks good (except for the merge-conflict).

Fixed.

**Pull request merged**